pkg/services/rendering/interface.go

```diff
@@ -26,15 +34,34 @@ type Opts struct {
 	Headers           map[string][]string
 }
 
+type CSVOpts struct {
+	Timeout         time.Duration
+	OrgId           int64
+	UserId          int64
```

Comment:
Go and initialisms in consistent case etc:
```suggestion
	OrgID           int64
	UserID          int64
```